content/develop/connect/insight/install-and-configure/_index.md

```diff
@@ -0,0 +1,14 @@
+---
+categories:
+- docs
+- operate
+- redisinsight
+description: 'Install and configure desktop Redis Insight or Redis Insight on Docker.
```

Comment:
```suggestion
description: Install and configure desktop Redis Insight or Redis Insight on Docker.
```

Reply:
I've removed @mich-elle-luna's comment. The leading single quote is required here.